### DISCUSSION TITLE
Default values for estate lists

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#441] [_Discarded pitches]
### Current state
For forms, it is possible to set a default value. For example, marketing type "Purchase".  Unfortunately, it is not possible to set this for property lists which are used to generate the property search module as shown here: https://wp-plugin-demo.onoffice.com/de/immobilien/immobilienliste/

Therefore we can not choose a default value for above mentioned property search module.

Current standard value = "Please select"

### Desired state

- Make default values available in estate lists, the same as in forms so the standard value can easily be changed to e.g. "Rent" or "Purchase"

![default_values_forms-1](https://user-images.githubusercontent.com/88546396/217253882-8ed22600-e145-47a5-a0ed-aba9c5d1932f.jpg)
![no_default_values_estatelist](https://user-images.githubusercontent.com/88546396/217253885-b5e9e8d9-3355-4fd4-87c5-1859c0801f07.PNG)

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long this will approx. take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need 5 days to fix and test it.
